src/gas/gas-util.ts

```diff
@@ -113,9 +111,9 @@ export async function fetchLegacyGasPriceEstimates(
 export async function fetchEthGasPriceEstimate(
   ethQuery: any,
 ): Promise<EthGasPriceEstimate> {
-  const gasPrice = await query(ethQuery, 'gasPrice');
+  const gasPrice = await query<string>(ethQuery, 'gasPrice');
   return {
-    gasPrice: weiHexToGweiDec(gasPrice).toString(),
+    gasPrice: weiHexToGweiDec(gasPrice),
```

Comment:
`weiHexToGweiDec` already returns a string, so there is no reason to convert this.